vars/main.yml

```diff
@@ -32,6 +35,12 @@ __snapshot_cmd: "{{ 'snapshot.py ' ~ snapshot_lvm_action ~ ' ' ~
   ('-s ' if snapshot_lvm_snapset_name else '') ~ ' ' ~
   (snapshot_lvm_snapset_name | quote
     if snapshot_lvm_snapset_name else '') ~ ' ' ~
+  ('-s ' if snapshot_lvm_mount_options else '') ~ ' ' ~
```

Comment:
using `-s` for both snapset name and mount options?  Maybe use `-S`?  Or some other letter?  I suggest `-M` which corresponds to `-m` for mount - `-m /dev/something -M foo,bar`

Reply:
-m is currently used for the mountpoint.  Since these are mount options, I went with -o.  

-O for the flag to mount the snapshot origin.